lib/charty/colors/xyz.rb

```diff
@@ -0,0 +1,125 @@
+require_relative "helper"
+
+require "numo/narray"
+
+module Charty
+  module Colors
+    XYZ2RGB = Numo::DFloat[
+      [  3.24096994190452134377, -1.53738317757009345794, -0.49861076029300328366 ],
+      [ -0.96924363628087982613,  1.87596750150772066772,  0.04155505740717561247 ],
+      [  0.05563007969699360846, -0.20397695888897656435,  1.05697151424287856072 ]
+    ]
+
+    RGB2XYZ = Numo::DFloat[
+      [  0.41239079926595948129,  0.35758433938387796373,  0.18048078840183428751 ],
+      [  0.21263900587151035754,  0.71516867876775592746,  0.07219231536073371500 ],
+      [  0.01933081871559185069,  0.11919477979462598791,  0.95053215224966058086 ]
+    ]
+
+    class XYZ < AbstractColor
+      include Helper
+
+      EPSILON = (6/29r)**3
+
+      KAPPA = (29/3)**3
+
+      def self.from_xyY(x, y, large_y)
+        large_x = large_y*x/y
+        large_z = large_y*(1 - x - y)/y
+        new(large_x, large_y, large_z)
+      end
+
+      def self.from_rgb(r, g, b)
```

Comment:
Is `RGB#to_xyz` not enough?

Reply:
I used XYZ class as a temporary place to put conversion methods of XYZ <-> RGB because XYZ color space is the hub of the color space conversion.  I will integrate the color space conversion features in a single module later.

BTW, adding `RGB#to_xyz` is good.  I forgot to add it.